packages/data-api/scripts/buildAnalyticsMaterializedView.sh

```diff
@@ -6,5 +6,9 @@ source .env
 
 cd scripts
 export PGPASSWORD=$DB_PASSWORD
-psql -p $DB_PORT -h $DB_URL -d $DB_NAME -U $DB_USER -f createAnalyticsMaterializedView.sql
-psql -p $DB_PORT -h $DB_URL -d $DB_NAME -U $DB_USER -f createAnalyticsTableIndexes.sql
+if [[ "$1" == "--force" || "$1" == "-f" ]]; then
```

Comment:
We can now force the analytics table to rebuild using `yarn workspace @tupaia/data-api build-analytics-table --force`
This will rebuild the table in the background, and then swap in the table once it's built, so tupaia should stay live the whole time.